src/execution_plan/ops/op_load_csv.c

```
@@ -17,23 +18,101 @@
+		ErrorCtx_RaiseRuntimeException(EMSG_INVALID_CSV_URI);
 		return false;
 	}
 
-	return true;
+	// supported URIs: file://
+	const char *csv_uri = op->uri.stringval;
+	static const char* URIS[1] = {"file://"};
+
+	// make sure uri is supported
+	for(int i = 0; i < 2; i++) {
+		const char *uri = URIS[i];
```

cppcheck (2.10-2)

[error] Array 'URIS[1]' accessed at index 1, which is out of bounds.

(arrayIndexOutOfBounds)